labs/private-connectivity/README.MD

```diff
@@ -0,0 +1,26 @@
+# APIM ❤️ OpenAI
+
+## [Private Connectivity Lab](private-connectivity.ipynb)
+
+![architecture](architecture.png)
+
+Playground to try the private connectivity [private connectivity functionality of APIM](https://learn.microsoft.com/en-us/azure/api-management/private-endpoint?tabs=v2) to a list of Azure OpenAI private endpoints.
+
+
+### Prerequisites
+
+- [Python 3.12 or later version](https://www.python.org/) installed
+- [VS Code](https://code.visualstudio.com/) installed with the [Jupyter notebook extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) enabled
+- [Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli) installed
+- [An Azure Subscription](https://azure.microsoft.com/free/) with Contributor permissions
+- [Access granted to Azure OpenAI](https://aka.ms/oai/access) or just enable the mock service
```

Comment:
I think we have moved on from mock services since AI services are now ubiquitously available.